docs/rhel-custom-image.md

```diff
@@ -0,0 +1,57 @@
+# RedHat Enterprise Linux
+
+Cloud providers which are listed below, support using rhel as an operating system option: 
+
+- AWS 
+- Azure
+- GCE
+- KubeVirt
+- Openstack
+- vSphere
+
+####  AWS:
+For amazon web service cloud provider, one the user applies for Gold Accesses Images using their RedHat Customer Portal, their request 
+will be reviewed and validated. Afterwards, new images will be added to the aws account under EC2-> Images-> AMIs-> Private Images.
+Once the images are available in the aws account, the image id for rhel(at the moment we only support 8.x.x versions) should be then 
+added to the `MachineDeployment` spec to the field `ami`.
+
+####  Azure
+Same for aws, RedHat supports Azure and provides images for it. Once those images are available, the field `imageID` in the 
```

Comment:
I think should be `Same as for AWS`, but I rather write something like:
```
RedHat provides images for Azure, [documentation](https://access.redhat.com/articles/uploading-rhel-image-to-azure) is available on RH customer portal.
```

Reply:
```
The `MachineDeployment` field `.spec.template.spec.providerSpec.value.cloudProviderSpec.imageID` should reference the ID of the uploaded VM.
```

Reply:
> The `MachineDeployment` field `.spec.template.spec.providerSpec.value.cloudProviderSpec.imageID` should reference the ID of the uploaded VM.

I think I will use this phrase for all places where I report the usage of the MachineDeployment :)